### ISSUE TITLE
About implementation of Batchnorm layer

### ISSUE THREAD

**Issue by linhlpv** [dmitrysarov/LRP_decomposition#1]
Hi, thank you for your great works !
I have read the tutorial about simple implementation of LRP in pytorch of heatmap at the website [tutorial](http://www.heatmapping.org/tutorial/). But they did not implement for Batchnorm layer. I find your implementation has it but I dont understand it much. Could you explain how to implement your implementation of Batchnorm layer in LRP.
Thank you again !
